js/main.js

```diff
@@ -85,118 +85,121 @@ function createPokemonDetail(id) {
   xhr.open('GET', 'https://pokeapi.co/api/v2/pokemon-species/' + id);
   xhr.responseType = 'json';
   xhr.addEventListener('load', function () {
-    var $pokemonDetailsBorder = document.createElement('div');
-    $pokemonDetailsBorder.setAttribute('class', 'pokemon-details-border height-test');
+    renderPokemonDetails(xhr);
+  });
+  xhr.send();
+}
 
-    var $divRow = document.createElement('div');
-    $divRow.setAttribute('class', 'row');
+function renderPokemonDetails(xhr) {
```

Comment:
You should change the parameter to something like `responseData` and pass `xhr.response` as the argument.